requirements.txt

```diff
@@ -3,6 +3,9 @@ scrapy
 # for s3:// schemes
 botocore
 
+# for scrapinghub collection
+scrapinghub
```

Comment:
Not sure why you're keeping a requirements.txt and also specifying requirements in setup.py. It's always better to keep a single source of truth.

Reply:
requirements.txt for development and setup.py for pip install